pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json

```diff
@@ -1,8492 +1,8659 @@
 {
-  "alpine": [
```

Comment:
what happened with the whitespace in this file?

Reply:
oh, not sure, I opened it and edited it in VS, so perhaps it corrected the spacing. I can revert that if we want to.

Reply:
These 2 json file should be generated using the `/p:UpdateRuntimeFiles=true` property. Are you using that?

https://github.com/dotnet/corefx/tree/master/pkg/Microsoft.NETCore.Platforms#adding-a-new-os

Reply:
Ok, I didn't know you could do that, I just did and updated the PR.